src/Commands/core/ArchiveDumpCommands.php

```diff
@@ -237,6 +242,74 @@ private function createManifestFile(array $options): void
         );
     }
 
+
+    /**
+     * Converts symlinks to the linked files/folders for an archive.
+     *
+     * @param bool $convert_symlinks
+     *  Whether to convert all symlinks.
+     * @param string $archivePath
+     *   The file path of the archive.
+     *
+     * @return void
+     */
+    public function convertSymlinks(
+        $convert_symlinks,
+        $archivePath
+    ): void {
```

Comment:
See https://github.com/drush-ops/drush/pull/5992/files#r1619382407, not resolved yet.

Reply:
Parameter types added